website/content/docs/api-gateway/consul-api-gateway-install.mdx

```diff
@@ -38,7 +40,7 @@ Ensure that the environment you are deploying Consul API Gateway in meets the re
      enabled: true
    apiGateway:
      enabled: true
-     image: hashicorp/consul-api-gateway:0.2.0
+     image: hashicorp/consul-api-gateway:VERSION
```

Comment:
Hmm...this gets a bit tricky as the user has to actually replace the value. If they use `VERSION` then they will get an error. 

We should use the highlight function and call out to update this with the version.